src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId)
+  || tabs[0];
+
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${currentTab.title}`}
+      </h1>
```

Comment:
Please check requirments and move h1 to App component
![image](https://github.com/mate-academy/react_tabs-js/assets/54936044/1c2c9336-01d7-4617-bbee-660c60fcd807)